### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by jbesraa**
issues:
- hover on the home icon will changes the bg. tried alot of ways to change it, no success. same to "Questions" at the sidebar.

UPDATE: Resolved

### CODE REVIEW

**Comment by des-des** [comment]
@esraajb if this is not in progress, is it ready for review?

**Review by njsfield — APPROVED** [review]

**Review by denesnori — APPROVED** [review]

**Review by njsfield — CHANGES_REQUESTED** [review]
Refactoring before merging

**Comment by des-des** [comment]
@njsfield @denesnori is this good to go? I see @njsfield has not approved

**Review by njsfield — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+<body>
+  <nav>
+    <h1 class="brand-logo center">SDMAPP</h1>
+    <ul class="right">
+      <li><a class="no-hover" href=#><i class="material-icons">home</i></a></ul>
+    <ul id="slide-out" class="side-nav" class="collection">
+        <li><a><h4>Questions</h4></a></li>
+      <li><a>What will this involve?</a></li>
+      <li class="divider"></li>
+      <li><a href="#!" id="1">How will this help me get better?</a></li>
```
Comment (des-des) [inline index.html:20]: `1` is ad name for id!

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+
+
+.no-hover:hover {
+  background-color: inherit !important;
+}
+
+/* .no-hover ovverides this default rule
+.nav ul a:hover {
```
Comment (des-des) [inline css/style.css:25]: :fire:

**Review by des-des — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (jbesraa) [reply index.html:20]: this was edited in the other PR i made #15

File: css/style.css (hunk 2 above)
Comment (jbesraa) [reply css/style.css:25]: this was edited in the other PR i made #15

**Review by des-des — APPROVED** [review]

**Pull request merged**